var/spack/repos/builtin/packages/py-pydv/package.py

```diff
@@ -7,13 +7,22 @@
 
 
 class PyPydv(PythonPackage):
-    """PDV is a 1D graphics and data analysis tool, heavily based on the
-    ULTRA plotting tool"""
+    """PyDV is a 1D graphics and data analysis tool, heavily based on the ULTRA plotting tool."""
 
     homepage = "https://github.com/griffin28/PyDV"
-    url = "https://github.com/griffin28/PyDV/archive/pydv-2.4.2.tar.gz"
+    # url = "https://github.com/griffin28/PyDV/archive/pydv-3.1.15.tar.gz"
+    pypi = "llnl-pydv/llnl_pydv-3.1.15.tar.gz"
+    git = "https://github.com/griffin28/PyDV"
+    tags = ["pydv-3.1.15"]
```

Comment:
Tags are designed to make it easy to search for common package (e.g., build tools). This tag is specific to a single version of a single package, so it isn't very useful.